X509-XWING-2024.asn

```diff
@@ -0,0 +1,58 @@
+XWing-KEM-2024
+   { iso(1) member-body(2) us(840) rsadsi(113549) pkcs(1)
+     pkcs-9(9) smime(16) modules(0) id-mod-XWing-kem-2024(TBD) }
+
+DEFINITIONS IMPLICIT TAGS ::= BEGIN
+
+-- EXPORTS ALL;
```

Comment:
Why is this commented out?

Reply:
Older ASN.1 compilers do not handle it, and it is the default behaviour.